server/Game.js

```diff
@@ -171,17 +158,33 @@ export default class {
     }
   }
 
+  getShuffledCandidates = (originalArray, cycles = MAX_CANDIDATES) => {
+    const shuffledArray = [...originalArray];
+    const cycleLimit = (shuffledArray.length - 1) - cycles;
+    
+    for (let i = shuffledArray.length - 1; i > 0 || i > cycleLimit; i--) {
```

Comment:
Hey, this should be `&&` instead of `||`.

```js
for (let i = shuffledArray.length - 1; i > 0 && i > cycleLimit; i--) {
   // ...
}
```

Sorry, my bad! 🙈

Reply:
You can just push another commit to fix this :-)